src/components/clusterConfiguration/NetworkConfiguration.tsx

```diff
@@ -81,30 +82,14 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
           isRequired
         />
       )}
-      <FormGroup fieldId="networkConfigurationType" label="Network Configuration">
-        <Radio
-          id="networkConfigurationTypeBasic"
-          name="networkConfigurationType"
-          isChecked={!isAdvanced}
-          value="basic"
-          onChange={toBasic}
-          label="Basic"
-          description="Use default networking options."
-          isLabelWrapped
-        />
-        <Radio
-          id="networkConfigurationTypeAdvanced"
-          name="networkConfigurationType"
-          value="advanced"
-          isChecked={isAdvanced}
-          onChange={() => setAdvanced(true)}
-          label="Advanced"
-          description="Configure a custom networking type and CIDR ranges."
-          isLabelWrapped
-        />
-      </FormGroup>
       <BasicNetworkFields cluster={cluster} hostSubnets={hostSubnets} />
-      {isAdvanced && <AdvancedNetworkFields />}
+      <CheckboxField
```

Comment:
It would be better to leave the checkbox outside of the Formik form values so it does not have an effect on 'validate and  save' button. Just render `Checkbox` component here and onChange do `setAdvanced` to toggle show/hide of the 'AdvancedNetworkConfiguration`. + execute `toBasic` if it got unchecked - just reset the values to defaults instead of initial values.

The initial value for `isAdvanced` can be calculated using `isAdvConf`

Reply:
It's a bit problematic to leave it outside of formik since then it won't remember the checkbox value if the page refreshes along with the changed values.

Reply:
Unless I misunderstood, the checkbox value should be initialised like this:
```
const [isAdvanced, setAdvanced] = React.useState(isAdvConf(cluster));
```

Reply:
done. please review.